_pages/docs.md

```diff
@@ -51,9 +51,37 @@ repository](http://github.com/proxyapps/proxyapps.github.io) of the proxy apps
 website.
 2. Lets say your new proxy application is called "proxyapp". Based on the
 apps.template.json file in the top level directory, create a new file
-apps-jsons/proxyapp.json and fill in the details.
+proxy-desc/proxyapp.json and fill in the details. Remove item, which do not
+apply. In principle, any valid json is fine, which can be
+checked [here](https://jsonlint.com/). Newline characters are not allowed in
+json, but one can add multiple by:
+```json
+[
+  { 
+    ...
+    "longdesc" : ["line1",
+                  "line2",
+                  "line3"]
+  }
+]     
+```
```

Comment:
@bhatele: Does our website support that?

Reply:
not sure